### PR TITLE
Add option to control element collision

### PR DESCRIPTION
**Pull request by CrosRoad95**

### CODE REVIEW

**Comment by CrosRoad95** [comment]
@Patrick2562 review

**Comment by gptrk0** [comment]
It would be nice to make it more universal and be able to switch more things on and off. _(interior/dimension change, ...)_
The function could be `setConfigOption(name, value)` / `getConfigOption(name)`.

**Comment by CrosRoad95** [comment]
Updated, please review.

File: client.lua [hunk 1]
```diff
@@ -32,7 +32,14 @@ pAttach = {
     inStreamPeds              = {},
     preparedToRenderInstances = {},
     pedsProcessedAdded        = false,
+    options                   = {
+        toggleCollision = true,
+    },
 
+    setConfigOption = function(self, name, value)
+        self.options[name] = value
```
Comment (Fernando-A-Rocha) [inline client.lua]: Add assert for name and value parameters

**Review by Fernando-A-Rocha — CHANGES_REQUESTED** [review]

**Comment by CrosRoad95** [comment]
Are those two asserts i added good enough?

**Comment by CrosRoad95** [comment]
@Fernando-A-Rocha

**Review by Fernando-A-Rocha — APPROVED** [review]
Yes all good Crossy 👌🏼

**Comment by CrosRoad95** [comment]
So when it get merged? @Patrick2562

**Comment by Patrick2562** [comment]
next week, i don't have time to check it in-game and I would like to make some changes before the next release

**Comment by gptrk0** [comment]
> So when it get merged? @Patrick2562

next week, i don't have time to check it in-game and I would like to make some changes before the next release

**Pull request merged**